src/loaders/workspace_downloader/workspace_downloader.py

```diff
@@ -414,7 +429,10 @@ def main():
             conf,
             loader_common_names.OBJECTS_NAME_ASSEMBLY,
         )
-        assembly_genome_map = _assembly_genome_lookup(genome_objs)
+        assembly_genome_map, duplicate_map = _assembly_genome_lookup(genome_objs)
+        if duplicate_map:
+            with open(os.path.join(output_dir, GENOME_DUPLICATE_FILE), "w") as outfile:
```

Comment:
At this point could you
* sort each of the lists in the missing dict by the save date
* print out a message saying something along the lines of that duplicates were found, the latest was kept, and the duplicates are in a file at `<path>`

Reply:
Updated. Currently is in ascending order.

Reply:
:+1: